### PR TITLE
i18n SC-4762 firstLogin

### PR DESCRIPTION
**Pull request by abralurrahman**
- [x] Link zum Ticket https://ticketsystem.schul-cloud.org/browse/SC-4762

### CODE REVIEW

File: views/firstLogin/firstLoginExistingUser.hbs [hunk 1]
```diff
@@ -9,16 +9,16 @@
...
       {{> "firstLogin/partials/welcome_existing"}}
 
-      <p>Bitte wähle: Ich bin...</p>
+      <p>{{$t "firstLogin.text.pleaseChoose" }}</p>
       <div class="form-group">
           <fieldset>
-              <input type="radio" id="reg-u14" name="student-age" value="u14"> <label for="reg-u14">unter 14 Jahre alt.</label><br/>
+              <input type="radio" id="reg-u14" name="student-age" value="u14"> <label for="reg-u14">{{$t "firstLogin.label.under14YearsOld" }}</label><br/>
               <input type="radio" id="reg-1415" name="student-age" value="1415"> <label for="reg-1415">zwischen 14 und {{ add CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS -1 }} Jahre alt.</label><br>
               <input type="radio" id="reg-16" name="student-age" value="16"> <label for="reg-16">ab {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} Jahre alt.</label>
```
Comment (DevSchmidtchen) [inline views/firstLogin/firstLoginExistingUser.hbs:17]: Missing translations

File: views/firstLogin/partials/language-hint.hbs [hunk 2]
```diff
@@ -1 +1 @@
-<p class="mt-2 explanation nomargin">* Sprachlicher Hinweis: Für bessere Lesbarkeit verwenden wir in der {{@root.theme.short_title}} die männliche Form (z.B. „Schüler“, „Lehrer“). Es sind stets alle Personen unabhängig vom Geschlecht gemeint. Die {{@root.theme.short_title}} ist für alle Menschen da, unabhängig von Herkunft, Einschränkungen, Geschlecht und sexueller Orientierung. 🌈🎉🦄 </p>
\ No newline at end of file
+<p class="mt-2 explanation nomargin">* {{$t "firstLogin.text.linguisticNote" }} {{@root.theme.short_title}} {{$t "firstLogin.text.theMaleForm" }} {{@root.theme.short_title}} {{$t "firstLogin.text.isThereForEveryone" }} 🌈🎉🦄 </p>
```
Comment (DevSchmidtchen) [inline views/firstLogin/partials/language-hint.hbs:1]: Interpolation would be better

File: views/firstLogin/partials/welcome_existing.hbs [hunk 3]
```diff
@@ -1,5 +1,5 @@
-<p>Herzlich willkommen zurück in der {{@root.theme.short_title}}, {{@root.currentUser.firstName}} {{@root.currentUser.lastName}},</p>
+<p>{{$t "firstLogin.welcome.text.welcomeBackTothe" }} {{@root.theme.short_title}}, {{@root.currentUser.firstName}} {{@root.currentUser.lastName}},</p>
```
Comment (DevSchmidtchen) [inline views/firstLogin/partials/welcome_existing.hbs:1]: Interpolation

File: views/firstLogin/sections/consent.hbs [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <section data-panel="section-{{section-nr}}" {{#if submit-page}}class="submit-page"{{/if}}>
-  <h2 class="h4">Einwilligungserklärung</h2>
-  <p>Bitte bestätige folgende Einwilligungserklärungen, damit du die {{@root.theme.title}} nutzen kannst.</p>
-  <p>Ich, <b>{{@root.currentUser.firstName}} {{@root.currentUser.lastName}}</b>, erkläre folgendes:</p>
+  <h2 class="h4">{{$t "firstLogin.global.heading.declarationOfConsent" }}</h2>
+  <p>{{$t "firstLogin.consent.text.pleaseConfirmTheFollowingDeclarationOfConsent" }} {{@root.theme.title}} {{$t "firstLogin.global.text.canUse" }}</p>
+  <p>{{$t "firstLogin.consent.text.i" }} <b>{{@root.currentUser.firstName}} {{@root.currentUser.lastName}}</b>, {{$t "firstLogin.consent.text.explainTheFollowing" }}</p>
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/consent.hbs:4]: Interpolation

File: views/firstLogin/sections/consent_updates.hbs [hunk 5]
```diff
@@ -1,26 +1,25 @@
 <section data-panel="section-{{section-nr}}" {{#if submit-page}}class="submit-page" {{/if}}>
 
-  <h2 class="h4">Wichtige Information{{#ifCond @root.updatedConsents.total '>' 1}}en{{/ifCond}}</h2>
-  <a style="margin-bottom: 20px;">Für ein noch besseres Lernerlebnis wird die {{@root.theme.title}} stetig
-    weiterentwickelt. Im Zuge dessen hat sich
-    {{#ifCond @root.updatedConsents.termsOfUse.total '>' 0}}die <a href="{{@root.theme.documents.specificFiles.termsOfUseExemplary}}" target="_blank">Nutzungsordnung</a>{{/ifCond}}
+  <h2 class="h4">{{$t "firstLogin.consent.heading.importantInformation" }} {{#ifCond @root.updatedConsents.total '>' 1}}en{{/ifCond}}</h2>
+  <a style="margin-bottom: 20px;">{{$t "firstLogin.consent.text.forAnEvenBetterLearningExperience" }}  {{@root.theme.title}} {{$t "firstLogin.consent.text.constantlyEvolving" }}
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/consent_updates.hbs:4]: Interpolation

File: views/firstLogin/sections/consent_updates.hbs [hunk 6]
```diff
@@ -1,26 +1,25 @@
...
     {{/ifCond}}
-    erweitert oder spezifisch für deine Schule angepasst.
+    {{$t "firstLogin.consent.text.expanded" }}
   </p>
   <div id="consentUpdateFields">
     {{#if @root.updatedConsents.haveBeenUpdated}}
 
     {{#each @root.updatedConsents.all.data}}
-    <h2 class="h4">Zusammenfassung der Erweiterung in der {{this.visualType}}</a> zum
+    <h2 class="h4">{{$t "firstLogin.consent.heading.summaryOfTheExtension" }} {{this.visualType}}</a> {{$t "firstLogin.consent.heading.toThe" }}
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/consent_updates.hbs:22]: Interpolation

File: views/firstLogin/sections/email.hbs [hunk 7]
```diff
@@ -1,11 +1,11 @@
...
+  <h2 class="h4">{{$t "firstLogin.email.heading.howCanWeContactYou" }}</h2>
+  <p>{{$t "firstLogin.email.text.pleaseCheckYourEmail" }}</p>
   <div class="row">
     <div class="form-group col-md-6 col-sm-12">
         <label>E-Mail:</label>
         <input class="form-control linked" type="email" name="student-email" placeholder="schueler@schul-cloud.org" value="{{@root.currentUser.email}}" readonly required>
     </div>
   </div>
-  <p>Sollte die E-Mail-Adresse nicht die richtige sein, wende dich bitte an deinen {{@root.theme.short_title}}-Administrator.</p>
+  <p>{{$t "firstLogin.email.text.ifTheEmailAddressIsNotTheCorrectOne" }} {{@root.theme.short_title}}-Administrator.</p>
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/email.hbs:10]: Interpolation

File: views/firstLogin/sections/parent_consent.hbs [hunk 8]
```diff
@@ -1,6 +1,6 @@
 <section data-panel="section-{{section-nr}}" {{#if submit-page}}class="submit-page"{{/if}}>
-  <h2 class="h4">Einwilligungserklärung</h2>
-  <p>Bitte bestätigen Sie folgende Einwilligungserklärungen, damit Ihr Kind die Vorteile der Schul-Cloud nutzen kann:</p>
-  <p>Ich, <b data-from="parent_firstName"></b> <b data-from="parent_lastName"></b>, bin erziehungsberechtigt für {{@root.currentUser.firstName}} {{@root.currentUser.lastName}} und erkläre Folgendes:</p>
+  <h2 class="h4">{{$t "firstLogin.global.heading.declarationOfConsent" }}</h2>
+  <p>{{$t "firstLogin.parent.text.pleaseConfirmTheFollowingDeclaration" }} </p>
+  <p>Ich, <b data-from="parent_firstName"></b> <b data-from="parent_lastName"></b>, {{$t "firstLogin.parent.text.amLegalGuardianFor" }}  {{@root.currentUser.firstName}} {{@root.currentUser.lastName}} {{$t "firstLogin.parent.text.andExplainTheFollowing" }} </p>
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/parent_consent.hbs:4]: Interpolation

File: views/firstLogin/sections/parent_data.hbs [hunk 9]
```diff
@@ -1,24 +1,24 @@
 <section data-panel="section-{{section-nr}}" {{#if submit-page}}class="submit-page"{{/if}}>
-  <h2 class="h4">Ihre Daten</h2>
-  <p>Für Schüler unter {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} Jahren bedarf ein Erziehungsberechtigter in die Nutzung der {{@root.theme.short_title}} einwilligen.</p>
-  <p>Dazu benötigen wir folgende Angaben von Ihnen:</p>
+  <h2 class="h4">{{$t "firstLogin.parent.heading.yourData" }}</h2>
+  <p>{{$t "firstLogin.parent.global.text.forStudentsUnder" }} {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} {{$t "firstLogin.parent.text.aParentOrLegalGuardianMystUse" }} {{@root.theme.short_title}} {{$t "firstLogin.parent.text.toConsent" }}</p>
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/parent_data.hbs:3]: Interpolation

File: views/firstLogin/sections/parent_data.hbs [hunk 10]
```diff
@@ -1,24 +1,24 @@
...
+  <p>{{$t "firstLogin.parent.text.forThisWeNeedTheFollowingInformationFromYou" }}</p>
   <div class="row">
     <div class="form-group col-md-6 col-sm-12">
-      <label>Vorname: *</label>
+      <label>{{$t "firstLogin.parent.global.label.firstName" }} *</label>
       <input class="form-control linked" name="parent_firstName" type="text" placeholder="Ihren Vornamen hier eingeben" required/>
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/parent_data.hbs:8]: placeholder not translated

File: views/firstLogin/sections/parent_data.hbs [hunk 11]
```diff
@@ -1,24 +1,24 @@
...
   </div>
 
   <div class="form-group">
-    <label>Ihre E-Mail-Adresse * <small>(Darf nicht gleich der E-Mail-Adresse Ihres Kindes sein)</small></label>
+    <label>{{$t "firstLogin.parent.global.label.yourEmailAddress" }} * <small>{{$t "firstLogin.parent.global.label.mustNotBeYourChildsEmailAddress" }}</small></label>
     <input class="form-control" type="email" name="parent_email" required placeholder="eltern@schul-cloud.org" autocomplete />
   </div>
   <p class="explanation nomargin">
-    Wir benötigen eine E-Mail-Adresse, damit Ihre Einwilligung in die Nutzung der {{@root.theme.short_title}} rechtlich gültig ist. Die {{@root.theme.short_title}} gibt die Adresse nicht weiter. Sie erhalten keine unerwünschten Mails.
+    {{$t "firstLogin.parent.text.weNeedAnEmailAddress" }} {{@root.theme.short_title}} {{$t "firstLogin.parent.text.islegallyValidThe" }} {{@root.theme.short_title}} {{$t "firstLogin.parent.text.doesNotPassOnTheAddress" }}
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/parent_data.hbs:21]: Interpolation

File: views/firstLogin/sections/parent_data.hbs [hunk 12]
```diff
@@ -1,24 +1,24 @@
...
     <br/>
     Zusätzlich planen wir die Entwicklung eines Eltern-Zugangs, über den Sie mit dem Klassenlehrer kommunizieren und mit dem Sie die Lernfortschritte Ihres Kindes nachvollziehen können. Sobald diese Funktionen zur Verfügung stehen, fragen wir Sie, ob wir Ihre E-Mail-Adresse dafür verwenden dürfen.
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/parent_data.hbs:23]: Missing translation

File: views/firstLogin/sections/parent_intro.hbs [hunk 13]
```diff
@@ -1,6 +1,6 @@
 <section data-panel="section-{{section-nr}}" class="parent_intro" data-next-label="als Erziehungsberechtigter fortfahren">
-  <h2 class="h4">Einverständniserklärung eines Erziehungsberechtigten benötigt</h2>
-  <p>Da du unter {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} Jahre alt bist, wird die Einverständniserklärung eines Elternteils benötigt, damit du die {{@root.theme.short_title}} nutzen kannst.
-    Die nächsten Schritte müssen von einem Erziehungsberechtigten durchgeführt werden.</p>
-  <a href="/logout" class="btn btn-secondary">Beenden und später fortfahren</a>
+  <h2 class="h4">{{$t "firstLogin.parent.heading.declarationOfConsentFromALegalGuardian" }}</h2>
+  <p>{{$t "firstLogin.parent.text.sinsceYouAreUnder" }} {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} {{$t "firstLogin.parent.text.yearsOldAparentsConsentFormRequired" }} {{@root.theme.short_title}} {{$t "firstLogin.global.text.canUse" }}
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/parent_intro.hbs:3]: Interpolation

File: views/firstLogin/sections/password.hbs [hunk 14]
```diff
@@ -1,20 +1,20 @@
...
-  <h2 class="h4">Du hast es fast geschafft...</h2>
+  <h2 class="h4">{{$t "firstLogin.heading.youAlmostMadeIt" }}</h2>
   {{#if @root.sso}}
     <p>Nach klicken des <u>Absenden</u>-Buttons kannst du dich wie gewohnt über das System deiner Schule (Moodle, iServ, ItsLearning etc.) in der {{@root.theme.title}} einloggen. </p>
   {{else}}
-    <p>Bitte ändere das Startpasswort. Notiere dir das neue Passwort gleich und bewahre es gut auf!</p>
+    <p>{{$t "firstLogin.text.pleaseChangeTheStartPassword" }}</p>
     <p>
       Mindestens 8 Zeichen mit Groß- und Kleinschreibung, davon jeweils mindestens eine Zahl und ein Sonderzeichen. <br/>
       Erlaubte Sonderzeichen sind: <code>!§$%&/()=?\;:,.#+*~</code>
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/password.hbs:9]: Missing translation

File: views/firstLogin/sections/thanks.hbs [hunk 15]
```diff
@@ -4,7 +4,7 @@
             <div class="content">
                 <img src="/images/firstLogin/welcome_screen.svg">
                 <hr class="mt-0">
-                    <h1 class="h2">Willkommen in der {{@root.theme.short_title}}</h1>
+                    <h1 class="h2">{{$t "firstLogin.welcome.text.welcomeTo" }} {{@root.theme.short_title}}</h1>
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/thanks.hbs:7]: Interpolation

File: views/firstLogin/sections/welcome.hbs [hunk 16]
```diff
@@ -1,7 +1,7 @@
 <section data-panel="section-{{section-nr}}" {{#if submit-page}}class="submit-page" {{/if}}>
-  <h2 class="h4">Herzlich willkommen in der {{@root.theme.short_title}}, {{@root.currentUser.firstName}} {{@root.currentUser.lastName}}</h2>
-  <p>Wir freuen uns, dass du die {{@root.theme.short_title}} nutzen möchtest.</p>
-  <p>Bitte gleiche kurz ein paar Daten ab, lege ein neues Passwort fest und dann kann es auch schon losgehen. 😊</p>
+  <h2 class="h4">{{$t "firstLogin.welcome.heading.welcomeToThe" }} {{@root.theme.short_title}}, {{@root.currentUser.firstName}} {{@root.currentUser.lastName}}</h2>
+  <p>{{$t "firstLogin.welcome.text.weArePleasedThatYou" }} {{@root.theme.short_title}} {{$t "firstLogin.welcome.text.wantToUse" }}</p>
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/welcome.hbs:3]: Interpolation

File: views/firstLogin/sections/welcome_14-15.hbs [hunk 17]
```diff
@@ -1,5 +1,5 @@
 <section data-panel="section-{{section-nr}}" {{#if submit-page}}class="submit-page"{{/if}}>
-  <h2 class="h4">Herzlich willkommen in der {{@root.theme.short_title}}, {{@root.currentUser.firstName}} {{@root.currentUser.lastName}} </h2>
+  <h2 class="h4">{{$t "firstLogin.welcome.heading.welcomeToThe" }} {{@root.theme.short_title}}, {{@root.currentUser.firstName}} {{@root.currentUser.lastName}} </h2>
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/welcome_14-15.hbs:2]: Interpolation

File: views/firstLogin/sections/welcome_14-15.hbs [hunk 18]
```diff
@@ -1,5 +1,5 @@
...
   <p>
       Wir freuen uns, dass du die {{@root.theme.title}} nutzen möchtest und mithilfst, gemeinsam den nächsten Schritt zur digitalen Unterstützung des Unterrichts zu gehen – Open Source, gemeinnützig und datenschutzkonform.
       Bitte überprüfe in den kommenden Schritten die für dich angegebene E-Mail-Adresse und stimme den Nutzungsbedingungen zu. Ein Erziehungsberechtigter hat der Nutzung bereits zugestimmt, jetzt bist du gefragt. Ändere anschließend dein Startpasswort, damit du die Vorteile der {{@root.theme.short_title}} nutzen kannst.
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/welcome_14-15.hbs:5]: Missing translation

File: views/firstLogin/sections/welcome_existing_geb14.hbs [hunk 19]
```diff
@@ -1,4 +1,4 @@
 <section data-panel="section-{{section-nr}}" class="confetti-wrapper{{#if submit-page}} submit-page{{/if}}">
-  <h2 class="h4">Herzlichen Glückwunsch, {{@root.currentUser.firstName}} {{@root.currentUser.lastName}}! 🎉🎈</h2>
-  <p>Du bist jetzt 14 Jahre alt. Bitte erkläre im folgenden Schritt dein Einverständnis zur Nutzung der {{@root.theme.title}}. Ab 14 Jahren ist dies zusätzlich zum bereits erklärten Einverständnis deiner Eltern nötig, damit du die {{@root.theme.short_title}} weiter nutzen kannst.</p>
+  <h2 class="h4">{{$t "firstLogin.welcome.heading.congratulations" }} {{@root.currentUser.firstName}} {{@root.currentUser.lastName}}! 🎉🎈</h2>
+  <p>{{$t "firstLogin.welcome.text.youAreNow14YearsOld" }} {{@root.theme.title}}. {{$t "firstLogin.welcome.text.fromTheAgeOF14" }} {{@root.theme.short_title}} {{$t "firstLogin.welcome.text.canContinueToUse" }}</p>
```
Comment (DevSchmidtchen) [inline views/firstLogin/sections/welcome_existing_geb14.hbs:3]: Interpolation

File: views/firstLogin/welcome.hbs [hunk 20]
```diff
@@ -6,19 +6,19 @@
 		<section class="empty-state">
 		  <img src="/images/empty-states/lehrer_klasse.svg" alt="Lehrer mit Klasse" class="image mx-auto d-block"/>
 		  <div class="text-wrapper mx-auto d-block">
-			<h2 class="text-center">Willkommen in der {{theme.title}}, {{currentUser.displayName}}</h2>
+			<h2 class="text-center">{{$t "firstLogin.welcome.text.welcomeTo" }} {{theme.title}}, {{currentUser.displayName}}</h2>
 			{{#unless (userHasRole "student")}}
 			  <h2 class="h5 text-muted text-center">
-				  Mit der {{theme.short_title}} Unterricht digital gestalten
+				  {{$t "firstLogin.welcome.text.withThe" }} {{theme.short_title}} {{$t "firstLogin.welcome.text.designLessonsDigitally" }}
```
Comment (DevSchmidtchen) [inline views/firstLogin/welcome.hbs:12]: Interpolation

File: views/firstLogin/sections/parent_data.hbs (hunk 10 above)
Comment (abralurrahman) [reply views/firstLogin/sections/parent_data.hbs:8]: Its Translated actually.

File: views/firstLogin/sections/password.hbs (hunk 14 above)
Comment (abralurrahman) [reply views/firstLogin/sections/password.hbs:9]: It will be translated as long text

File: views/firstLogin/sections/parent_consent.hbs [hunk 21]
```diff
@@ -1,6 +1,6 @@
 <section data-panel="section-{{section-nr}}" {{#if submit-page}}class="submit-page"{{/if}}>
-  <h2 class="h4">Einwilligungserklärung</h2>
-  <p>Bitte bestätigen Sie folgende Einwilligungserklärungen, damit Ihr Kind die Vorteile der HPI Schul-Cloud nutzen kann:</p>
-  <p>Ich, <b data-from="parent_firstName"></b> <b data-from="parent_lastName"></b>, bin erziehungsberechtigt für {{@root.currentUser.firstName}} {{@root.currentUser.lastName}} und erkläre Folgendes:</p>
+  <h2 class="h4">{{$t "firstLogin.global.heading.declarationOfConsent" }}</h2>
+  <p>{{$t "firstLogin.parent.text.pleaseConfirmTheFollowingDeclaration" }}</p>
+  <p>Ich, <b data-from="parent_firstName"></b> <b data-from="parent_lastName"></b>, {{$t "firstLogin.parent.text.amLegalGuardianFor" }} {{@root.currentUser.firstName}} {{@root.currentUser.lastName}} und erkläre Folgendes:</p>
```
Comment (dominikbruechner) [inline views/firstLogin/sections/parent_consent.hbs:4]: translation missing

File: views/firstLogin/sections/parent_data.hbs (hunk 11 above)
Comment (dominikbruechner) [reply views/firstLogin/sections/parent_data.hbs:21]: interpolation missing

File: views/firstLogin/sections/parent_data.hbs (hunk 10 above)
Comment (dominikbruechner) [reply views/firstLogin/sections/parent_data.hbs:8]: the placeolder Ihren Vornamen hier eingeben is not translated

File: views/firstLogin/sections/parent_data.hbs (hunk 12 above)
Comment (dominikbruechner) [reply views/firstLogin/sections/parent_data.hbs:23]: why are you resolving issues here, which are clearly not resolved? please fix

File: views/firstLogin/sections/password.hbs (hunk 14 above)
Comment (dominikbruechner) [reply views/firstLogin/sections/password.hbs:9]: can you please do it here?

File: views/firstLogin/sections/password.hbs [hunk 22]
```diff
@@ -1,20 +1,20 @@
 <section data-panel="section-{{section-nr}}" {{#if submit-page}}class="submit-page"{{/if}}>
-  <h2 class="h4">Du hast es fast geschafft...</h2>
+  <h2 class="h4">{{$t "firstLogin.heading.youAlmostMadeIt" }}</h2>
   {{#if @root.sso}}
     <p>Nach klicken des <u>Absenden</u>-Buttons kannst du dich wie gewohnt über das System deiner Schule (Moodle, iServ, ItsLearning etc.) in der {{@root.theme.title}} einloggen. </p>
```
Comment (dominikbruechner) [inline views/firstLogin/sections/password.hbs:4]: please add translation here

File: views/firstLogin/sections/thanks.hbs (hunk 15 above)
Comment (dominikbruechner) [reply views/firstLogin/sections/thanks.hbs:7]: please add interpolation here

File: views/firstLogin/welcome.hbs (hunk 20 above)
Comment (dominikbruechner) [reply views/firstLogin/welcome.hbs:12]: interpolation missing

File: views/firstLogin/sections/password.hbs (hunk 14 above)
Comment (abralurrahman) [reply views/firstLogin/sections/password.hbs:9]: sure!

File: views/firstLogin/sections/parent_intro.hbs [hunk 23]
```diff
@@ -1,6 +1,6 @@
 <section data-panel="section-{{section-nr}}" class="parent_intro" data-next-label="als Erziehungsberechtigter fortfahren">
```
Comment (dominikbruechner) [inline views/firstLogin/sections/parent_intro.hbs:1]: missing i18n

**Pull request merged**